doc/source/timeseries.rst

```diff
@@ -402,15 +392,29 @@ using various combinations of parameters like ``start``, ``end``,
 
    pd.bdate_range(start=start, periods=20)
 
-The start and end dates are strictly inclusive. So it will not generate any
```

Comment:
put a small ::warning box where you say ``cdate_range`` is deprecated, add a ref as well (then you can refer to it in whatsnew)

Reply:
in the appropriate location in this file